tools/migrations/convert_all_tables_to_innodb.py

```diff
@@ -0,0 +1,24 @@
+import array
+import mysql.connector
+
+def migration_name():
+	return "Converting DB engine for all tables from MyISAM to InnoDB"
+
+def check_preconditions(cur):
+	return
+
+def needs_to_run(cur):
+	cur.execute("SELECT * FROM INFORMATION_SCHEMA.TABLES WHERE TABLE_SCHEMA = 'tpzdb' AND ENGINE = 'MyISAM' LIMIT 1")
```

Comment:
Can't we get the table schema name from one of the conf files?

Reply:
We could, but querying guarantees that all tables get converted. The cost isn't important because it's a one-time conversion